src/PowerManifold.jl

```diff
@@ -530,22 +530,122 @@ function default_vector_transport_method(M::PowerManifold, t::Type)
     return default_vector_transport_method(M.manifold, eltype(t))
 end
 
-@doc raw"""
-    distance(M::AbstractPowerManifold, p, q)
+_doc_distance_pow = """
+    distance(M::AbstractPowerManifold, p, q, r::Real=2.0)
+    distance(M::AbstractPowerManifold, p, q, m::AbstractInverseRetractionMethod=LogarithmicInverseRetraction(), r::Real=2.0)
+
+Compute the distance between `q` and `p` on an [`AbstractPowerManifold`](@ref).
 
-Compute the distance between `q` and `p` on an [`AbstractPowerManifold`](@ref),
-i.e. from the element wise distances the Forbenius norm is computed.
+First, the componentwise distances are computed. These can be approximated using the
+`norm` of an [`AbstractInverseRetractionMethod`](@ref) `m`.
+Then, the `r`-norm of these elements is computed.
```

Comment:
```suggestion
Then, the `r`-norm of these elements is computed.

Note that the `r` argument is *not* passed recursively to components when the distance is computed.
```
Maybe that would make it clear?

Reply:
I am not so sure. In 98% of times “passing down” does not even make sense (that is outside of Product/Power of Power/Product – on Manifolds.jl maybe Euclidan).
I was of the impression that “of these elements” should be precise enough.

Reply:
To me passing-down is a sufficiently plausible alternative that we should make this point very clear in the docstring. Not passing it is of course perfectly fine, it's just a matter of making the documentation clear.

Reply:
Irgs, I was too fast an added this. Meh. My brain starts melting after about 3 hours of debugging corner cases I fear.

Reply:
The passing down reminds me of the “autobroadcast” in Matlab, where when you accidentally add a row to a column vector, you get the corresponding rank1-1matrix.
Expecially when it is context dependent whether that can happen, this is dangerous magic. We should not do that. And even getting it right would be challenging, For every distance/norm call we would have to check whether we can pass down r? Uff.

Reply:
Sorry, I just wanted to discuss this issue. Not passing it down is perfectly fine.

Reply:
And about documenting it right, I had hoped That is documented originally already right.
We do _elementwise_ the _classical_ distance. on that we do the R-norm

I would also prefer to document, _what_ we do and not _what we not do_ ;) Because what we not do, we would add a lot of things for sure as well. The method also does not make you dinner ;)